.github/workflows/run-tests.yml

```diff
@@ -20,10 +20,24 @@ on:
 
 jobs:
   run-tests:
-    name: Run tests job
+    name: Run tests job matrix
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
```

Comment:
thought: This will increase the visibility of any test flakiness.  This could be extremely annoying, but will force us to sort out flakiness.